diff -aurN paperwork-2.2.1.orig/openpaperwork-core/MANIFEST.in paperwork-2.2.1/openpaperwork-core/MANIFEST.in
--- paperwork-2.2.1.orig/openpaperwork-core/MANIFEST.in	2023-10-02 15:12:24.246949696 +0200
+++ paperwork-2.2.1/openpaperwork-core/MANIFEST.in	2023-10-02 15:16:36.916770704 +0200
@@ -1,4 +1,2 @@
-recursive-include src *.py *.mo
-recursive-include tests *
-include *.md
+recursive-include src *.py
 include LICENSE
diff -aurN paperwork-2.2.1.orig/openpaperwork-core/src/openpaperwork_core/l10n/python.py paperwork-2.2.1/openpaperwork-core/src/openpaperwork_core/l10n/python.py
--- paperwork-2.2.1.orig/openpaperwork-core/src/openpaperwork_core/l10n/python.py	2023-10-02 15:12:24.246949696 +0200
+++ paperwork-2.2.1/openpaperwork-core/src/openpaperwork_core/l10n/python.py	2023-10-02 15:17:20.357428088 +0200
@@ -68,16 +68,7 @@
         self.l10n_load('openpaperwork_core.l10n', 'openpaperwork_core')
 
     def l10n_load(self, python_package, text_domain):
-        path = self.core.call_success(
-            "resources_get_dir", python_package, 'out'
-        )
-        if path is None:
-            LOGGER.error(
-                "Failed to access ressources '%s/out'", python_package
-            )
-            return None
-
-        path = self.core.call_success("fs_unsafe", path)
+        path = None
 
         mo_file = gettext.find(text_domain, path)
         if mo_file is None:
diff -aurN paperwork-2.2.1.orig/openpaperwork-core/src/openpaperwork_core/resources/setuptools.py paperwork-2.2.1/openpaperwork-core/src/openpaperwork_core/resources/setuptools.py
--- paperwork-2.2.1.orig/openpaperwork-core/src/openpaperwork_core/resources/setuptools.py	2023-10-02 15:12:24.246949696 +0200
+++ paperwork-2.2.1/openpaperwork-core/src/openpaperwork_core/resources/setuptools.py	2023-10-02 15:18:14.862252893 +0200
@@ -28,15 +28,18 @@
         if not PKG_RESOURCES_AVAILABLE:
             return
 
-        try:
-            path = pkg_resources.resource_filename(pkg, filename)
-        except KeyError:
-            LOGGER.warning(
-                "Failed to find resource file %s/%s,"
-                " unknown to pkg_resources.",
-                pkg, filename
-            )
-            return None
+        if pkg == "paperwork_gtk.model.help.out":
+            path = "/usr/share/doc/paperwork-gtk/{}".format(filename)
+        else:
+            try:
+                path = pkg_resources.resource_filename(pkg, filename)
+            except KeyError:
+                LOGGER.warning(
+                    "Failed to find resource file %s/%s,"
+                    " unknown to pkg_resources.",
+                    pkg, filename
+                )
+                return None
 
         if not os.access(path, os.R_OK):
             LOGGER.warning(
diff -aurN paperwork-2.2.1.orig/openpaperwork-gtk/MANIFEST.in paperwork-2.2.1/openpaperwork-gtk/MANIFEST.in
--- paperwork-2.2.1.orig/openpaperwork-gtk/MANIFEST.in	2023-10-02 15:12:24.247949711 +0200
+++ paperwork-2.2.1/openpaperwork-gtk/MANIFEST.in	2023-10-02 15:19:07.988037633 +0200
@@ -1,4 +1,2 @@
-recursive-include src *.py *.glade *.xml *.css *.svg *.png *.mo
-recursive-include tests *
-include *.md
+recursive-include src *.py *.glade *.xml *.css *.svg *.png
 include LICENSE
diff -aurN paperwork-2.2.1.orig/paperwork-backend/MANIFEST.in paperwork-2.2.1/paperwork-backend/MANIFEST.in
--- paperwork-2.2.1.orig/paperwork-backend/MANIFEST.in	2023-10-02 15:12:24.247949711 +0200
+++ paperwork-2.2.1/paperwork-backend/MANIFEST.in	2023-10-02 15:19:34.332429674 +0200
@@ -1,4 +1,2 @@
-recursive-include src *.py *.mo *.json
-recursive-include tests *
-include *.markdown
+recursive-include src *.py *.json
 include LICENSE
diff -aurN paperwork-2.2.1.orig/paperwork-gtk/MANIFEST.in paperwork-2.2.1/paperwork-gtk/MANIFEST.in
--- paperwork-2.2.1.orig/paperwork-gtk/MANIFEST.in	2023-10-02 15:12:24.250949757 +0200
+++ paperwork-2.2.1/paperwork-gtk/MANIFEST.in	2023-10-02 15:19:59.702809581 +0200
@@ -1,5 +1,2 @@
-recursive-include src *.py *.glade *.xml *.css *.svg *.png *.pdf *.mo
-recursive-include tests *
-include *.markdown
-include example-paperwork.conf
+recursive-include src *.py *.glade *.xml *.css *.svg *.png *.pdf
 include COPYING
diff -aurN paperwork-2.2.1.orig/paperwork-shell/MANIFEST.in paperwork-2.2.1/paperwork-shell/MANIFEST.in
--- paperwork-2.2.1.orig/paperwork-shell/MANIFEST.in	2023-10-02 15:12:24.252949787 +0200
+++ paperwork-2.2.1/paperwork-shell/MANIFEST.in	2023-10-02 15:20:21.062130985 +0200
@@ -1,4 +1,2 @@
-recursive-include src *.py *.mo *.png
-recursive-include tests *
-include *.markdown
+recursive-include src *.py *.png
 include LICENSE
