diff -aurN paperwork-2.2.1.orig/openpaperwork-core/src/openpaperwork_core/__init__.py paperwork-2.2.1/openpaperwork-core/src/openpaperwork_core/__init__.py
--- paperwork-2.2.1.orig/openpaperwork-core/src/openpaperwork_core/__init__.py	2023-10-02 15:12:24.246949696 +0200
+++ paperwork-2.2.1/openpaperwork-core/src/openpaperwork_core/__init__.py	2023-10-02 15:13:11.242660248 +0200
@@ -46,12 +46,6 @@
     'openpaperwork_core.work_queue.default',
 ]
 
-if os.name != 'nt':
-    RECOMMENDED_PLUGINS += [
-        'openpaperwork_core.cmd.chkdeps',
-    ]
-
-
 def _(s):
     return gettext.dgettext('openpaperwork_core', s)
 
diff -aurN paperwork-2.2.1.orig/paperwork-gtk/src/paperwork_gtk/main.py paperwork-2.2.1/paperwork-gtk/src/paperwork_gtk/main.py
--- paperwork-2.2.1.orig/paperwork-gtk/src/paperwork_gtk/main.py	2023-10-02 15:12:24.250949757 +0200
+++ paperwork-2.2.1/paperwork-gtk/src/paperwork_gtk/main.py	2023-10-02 15:14:03.680452716 +0200
@@ -63,7 +63,6 @@
         'paperwork_gtk.actions.page.redo_ocr',
         'paperwork_gtk.actions.page.reset',
         'paperwork_gtk.cmd.import',
-        'paperwork_gtk.cmd.install',
         'paperwork_gtk.doc_selection',
         'paperwork_gtk.docimport',
         'paperwork_gtk.drawer.calibration',
@@ -158,7 +157,6 @@
         'paperwork_gtk.settings.scanner.settings',
         'paperwork_gtk.settings.stats',
         'paperwork_gtk.settings.storage',
-        'paperwork_gtk.settings.update',
         'paperwork_gtk.shortcuts.app.find',
         'paperwork_gtk.shortcuts.doc.new',
         'paperwork_gtk.shortcuts.doc.prev_next',
@@ -168,7 +166,6 @@
         'paperwork_gtk.shortcuts.page.edit',
         'paperwork_gtk.shortcutswin',
         'paperwork_gtk.sync_on_start',
-        'paperwork_gtk.update_notification',
         'paperwork_gtk.widget.flowlayout',
         'paperwork_gtk.widget.label',
     ]
diff -aurN paperwork-2.2.1.orig/paperwork-gtk/src/paperwork_gtk/model/help/screenshot.sh paperwork-2.2.1/paperwork-gtk/src/paperwork_gtk/model/help/screenshot.sh
--- paperwork-2.2.1.orig/paperwork-gtk/src/paperwork_gtk/model/help/screenshot.sh	2023-10-02 15:12:24.252949787 +0200
+++ paperwork-2.2.1/paperwork-gtk/src/paperwork_gtk/model/help/screenshot.sh	2023-10-02 15:14:35.084928700 +0200
@@ -39,6 +39,10 @@
 echo "Making screenshots ..."
 paperwork-gtk plugins add openpaperwork_core.interactive
 
+echo "Enable plugins disabled for building"
+paperwork-gtk plugins add paperwork_gtk.settings.update
+paperwork-gtk plugins add paperwork_gtk.update_notification
+
 paperwork-gtk << EOF
 wait()
 core.call_all("doc_open", "20990307_0000_00", "file://${WORKDIR}/20990307_0000_00")
